currency/code_test.go

```diff
@@ -456,6 +458,14 @@ func TestBaseCode(t *testing.T) {
 	}
 }
 
+func TestNewCodeFormatting(t *testing.T) {
+	require.True(t, NewCode("BTC").UpperCase)
+	require.False(t, NewCode("btc").UpperCase)
+	require.True(t, NewCode("BTC").Equal(NewCode("btc")))
+	require.True(t, NewCode("420").UpperCase)
+	require.False(t, NewCode("btc420").UpperCase)
+}
```

Comment:
Just seeking clarification, is it that you wish for digits to always be upper case? 
One can change that with
```go
	butts := NewCode("420")
	hi := butts.Lower()
	t.Log(hi.UpperCase)
```

```
    code_test.go:470: false
```

Reply:
[done](https://github.com/thrasher-corp/gocryptotrader/pull/1593/commits/ed3a1a7c9955f83d1bffc2e33dd1f9f574035cca)